src/components/TextField/TextField.tsx

```diff
@@ -442,6 +443,10 @@ class TextField extends React.PureComponent<CombinedProps, State> {
       this.buttonPressTimer = window.setTimeout(onChangeInterval, interval);
     };
     this.buttonPressTimer = window.setTimeout(onChangeInterval, interval);
+
+    addEventListener(document, 'mouseup', this.handleButtonRelease, {
+      once: true,
+    });
```

Comment:
Should this be torn down in `handleButtonRelease` if it will start listening again on subsequent button clicks?

Reply:
The `once` option will tear down the event after it runs one time.